XExtensionItem/XExtensionItemSourceApplication.m

```diff
@@ -9,7 +9,9 @@ @implementation XExtensionItemSourceApplication
 #pragma mark - Initialization
 
 - (instancetype)initWithAppNameFromBundle:(NSBundle *)bundle appStoreID:(NSNumber *)appStoreID {
-    return [self initWithAppName:bundle.infoDictionary[(NSString *)kCFBundleNameKey] appStoreID:appStoreID];
+    NSString *displayName = bundle.infoDictionary[@"CFBundleDisplayName"] ?: bundle.infoDictionary[(NSString *)kCFBundleNameKey];
```

Comment:
Maybe worth making it a constant ourselves just for future readers of the code, so they know we’re not just making it up. Idk.